src/component/Tooltip.tsx

```diff
@@ -90,82 +90,78 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Def
   useTranslate3d?: boolean;
 };
 
-export class Tooltip<TValue extends ValueType, TName extends NameType> extends PureComponent<
-  TooltipProps<TValue, TName>
-> {
-  static displayName = 'Tooltip';
-
-  static defaultProps = {
-    active: false,
-    allowEscapeViewBox: { x: false, y: false },
-    reverseDirection: { x: false, y: false },
-    offset: 10,
-    viewBox: { x1: 0, x2: 0, y1: 0, y2: 0 },
-    coordinate: { x: 0, y: 0 },
-    cursorStyle: {},
-    separator: ' : ',
-    wrapperStyle: {},
-    contentStyle: {},
-    itemStyle: {},
-    labelStyle: {},
-    cursor: true,
-    trigger: 'hover',
-    isAnimationActive: !Global.isSsr,
-    animationEasing: 'ease',
-    animationDuration: 400,
-    filterNull: true,
-    useTranslate3d: false,
-  };
+const tooltipDefaultProps = {
+  active: false,
+  allowEscapeViewBox: { x: false, y: false },
+  reverseDirection: { x: false, y: false },
+  offset: 10,
+  viewBox: { x: 100, y: 100, height: 50, width: 50 },
```

Comment:
What is the reasoning for these specific values? - as opposed to the existing ones?

Reply:
`{ x1: 0, x2: 0, y1: 0, y2: 0 }` - these are not valid values on `viewBox` - there is no such thing as x1, y1, x2, y2

As for the values I think these are the default viewBox values for the tooltip but I honestly forget. Will check

Reply:
I understand the change, but it seems an unrelated bug fix. I did not find these default values anywhere, I believe this might be changing behaviour. Please consider doing this change in a prefactoring PR, keeping the refactoring class --> function a pure refactoring without behavioural change.

Reply:
I will test the Tooltip behaviour on this PR. This is the only change that I suspect might be an issue / change behaviour.

Reply:
Hmm okay, I think I got these values from somewhere but I don't remember exactly where. Either way these x1, y1 etc. fields are incorrect and need addressed somewhere. I can ts-ignore them here and fix later